exes/ethercat/src/devices/beckhoff.cpp

```diff
@@ -0,0 +1,93 @@
+#include <fmt/format.h>
+
+#include <tfc/ec/devices/beckhoff/EL1xxx.hpp>
+#include <tfc/ec/devices/beckhoff/EL2xxx.hpp>
+#include <tfc/ipc.hpp>
+#include <tfc/ipc/details/dbus_server_iface.hpp>
+
+namespace tfc::ec::devices::beckhoff {
+
+// TODO discuss! I would like to remove the name of the device from the ipc channel name
```

Comment:
Elaborate your thoughts on this :smile:

Reply:
How would you name them?
Are you thinking about a per network digital IO list like plc's do
```
Q0.1
Q0.2
```

I don't really like that idea, then if you switch to something generic like 
```
DI16.{ethercat_id}.0
DI16.{ethercat_id}.1
```

Reply:
What I am thinking is something like `ethercat.def.slave3.input4` but let's discuss this in person. Will remove comment